cni/plugin.go

```diff
@@ -156,8 +156,8 @@ func (plugin *Plugin) printVersionInfo() error {
 	return nil
 }
 
-// LookupUser returns the UID for the given username, or the current user.
-func (plugin *Plugin) LookupUser(userName string) (int, error) {
+// LookupUser returns the UID and GID for the given username, or the current user.
+func (plugin *Plugin) LookupUser(userName string) (int, int, error) {
```

Comment:
This method doesn't really belong in the plugin class. We need to move this logic to another package. Not necessary for this PR, just making a note here.

Reply:
ok, let's do that separately, let me know if you want me to create an issue for it.